spec/models/user_spec.rb

```diff
@@ -161,13 +162,13 @@
     it(nil, probedock: { key: 'h066' }){ should validate_presence_of(:password) }
 
     it "should validate that the primary e-mail is among the user's e-mails", probedock: { key: 'rmzp' } do
-      user = build :user
+      user = build(:user)
       user.emails = []
       expect(user).not_to be_valid
     end
 
-    describe "with an existing user" do
-      let!(:user){ create :user }
+    describe 'with an existing user' do
+      let!(:user){ create(:user) }
       it(nil, probedock: { key: 'f9be952c7792' }){ should validate_uniqueness_of(:name) }
```

Comment:
Add a test to ensure that a human user can be created if a technical user exists with the same name.